platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/config/ConfigUtils.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.test.fixtures.config;
+
+import com.swirlds.common.config.StateCommonConfig;
+import com.swirlds.common.io.config.TemporaryFileConfig;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.config.api.ConfigurationBuilder;
+import com.swirlds.merkledb.config.MerkleDbConfig;
+import com.swirlds.virtualmap.config.VirtualMapConfig;
+
+public final class ConfigUtils {
+    private ConfigUtils() {}
+
+    public static final Configuration CONFIGURATION = ConfigurationBuilder.create()
```

Comment:
There is no javadoc in this class. Also, it seems to provide functionality for a very specific use case, so I think it needs to be moved and renamed accordingly

Reply:
This is coming from https://github.com/hashgraph/hedera-services/pull/15562, I picked it here in order to minimize the number of conflicts that would arise when the PRs get merged. 
The configuration related changes here were related to addressing [this comment](https://github.com/hashgraph/hedera-services/pull/14695#discussion_r1782018018) and that has increased the scope (i.e. files changed) of this PR significantly imo.
I would suggest this to be addressed in https://github.com/hashgraph/hedera-services/pull/15562 instead. cc @thenswan

Reply:
I am good to address it in #15562 that I opened